src/request_builder.rs

```diff
@@ -120,7 +120,10 @@ impl RequestBuilder {
         self
     }
 
-    /// Sets the body of the request.
+    /// Pass an `AsyncRead` stream as the request body.
```

Comment:
Maybe anything with implements `<link to body docs Into>`, ... "such as `AsyncRead`"?

Reply:
Not sure I quite follow -- can you maybe share what text you'd like to see here instead?

Reply:
```suggestion
    /// Sets the body of the request from any type with implements `Into<Body>`, for example, any type with is `AsyncRead`.
```
Perhaps?